### DISCUSSION TITLE
How to solve a biharmonic function on a triangular mesh

### DISCUSSION THREAD

**Discussion by qiqigit** [libigl/libigl#1943] [Q&A] [answered]
Hi all,

I am trying to solve a (scalar) biharmonic function defined on a triangular mesh with constraints (predefined values) on some of the vertices. ※The ultimate goal here is to extract an isoline with value 0.

I am wondering if/how I can do this with igl::harmonic.

I would appreciate any help with this. Thanks!

**Comment by alecjacobson** [comment, marked as answer]
yes, you can do that by passing k = 2
